modules/widgets/facebook-likebox.php

```diff
@@ -76,9 +76,21 @@ function widget( $args, $instance ) {
 		$title    = apply_filters( 'widget_title', $instance['title'] );
 		$page_url = set_url_scheme( $like_args['href'], 'https' );
 
-		$like_args['show_faces'] = (bool) $like_args['show_faces'] ? 'true' : 'false';
-		$like_args['stream']     = (bool) $like_args['stream'] ? 'timeline' : 'false';
-		$like_args['cover']      = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['show_faces']   = (bool) $like_args['show_faces'] ? 'true' : 'false';
+		$like_args['stream']       = (bool) $like_args['stream'] ? 'timeline' : 'false';
+		$like_args['cover']        = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['small_header'] = (bool) $like_args['small_header'] ? 'true' : 'false';
+
+		/**
+		 * Filter Facebook Likebox's widget call to action button
+		 *
+		 * @module widgets
+		 *
+		 * @since 8.2.4
+		 *
+		 * @param bool True value hides the call to action button
+		 */
+		$hide_cta = apply_filters( 'jetpack_fb_hide_cta', 'true' );
```

Comment:
Since it's a boolean, you'll want to remove the `'` to really keep a boolean and not a string.

```suggestion
		$hide_cta = apply_filters( 'jetpack_fb_hide_cta', true );
```

Reply:
done